css/amp.css

```diff
@@ -621,6 +621,20 @@ i-amphtml-scroll-container.amp-active, i-amp-scroll-container.amp-active {
   visibility: visible;
 }
 
+amp-list[load-more] > [load-more-button] > .i-amphtml-loader {
```

Comment:
not sure why this is showing up in my diff as new, these rules already exist in amp.css with the exception of the new `load-more-end`

https://github.com/ampproject/amphtml/blob/3123726fbe724e551b0ea4c8e83b34871d0f5822/css/amp.css#L803

Reply:
Actually yea these were all added due to the incorrect rebase. I've removed them all.